lib/ui/views/record/record_view.dart

```diff
@@ -178,7 +179,38 @@ class _RecordViewState extends State<RecordView> {
           : formattedText;
     } catch (e) {
```

Comment:
Here please make exception promperly catch example 400, 500, 401 , catch won't be able to identify issues with exact code